arches/app/models/resource.py

```diff
@@ -281,8 +281,7 @@ def save(self, **kwargs):
 
         if not self.principaluser_id and user:
             self.principaluser_id = user.id
-            # TODO: this is not updating update_fields in kwargs
-            # Fix when adding user argument to Resource.save().
+            add_to_update_fields(kwargs, "principaluser_id")
```

Comment:
I no longer plan to suggest the change in that TODO, so just rolling in the fix now.